CRM/Pledge/DAO/Pledge.php

```diff
@@ -38,6 +38,19 @@ class CRM_Pledge_DAO_Pledge extends CRM_Core_DAO {
    */
   public static $_log = TRUE;
 
+  /**
+   * Paths for accessing this entity in the UI.
+   *
+   * @var string[]
+   */
+  protected static $_paths = [
+    'add' => 'civicrm/pledge/add?action=add&context=standalone&reset=1',
+    'view' => 'civicrm/contact/view/pledge?id=[id]&cid=[contact_id]&action=view&reset=1',
+    'update' => 'civicrm/contact/view/pledge?id=[id]&cid=[contact_id]&action=update&reset=1',
+    'delete' => 'civicrm/contact/view/pledge?id=[id]&cid=[contact_id]&action=delete&reset=1',
+    'cancel' => 'civicrm/contact/view/pledge?id=[id]&cid=[contact_id]&action=detach&reset=1',
```

Comment:
This `"cancel"` action is nonstandard and doesn't show up in SearchKit. IMO it probably shouldn't because loading that page immediately cancels the pledge with no confirmation!
@aydun on second thought... maybe we shouldn't include this one at all.